src/main/java/uk/gov/hmcts/reform/wataskconfigurationapi/clients/CamundaServiceApi.java

```diff
@@ -27,19 +27,17 @@
 public interface CamundaServiceApi {
 
     @PostMapping(
-        value = "/decision-definition/key/{decisionTableName}-{jurisdiction}-{caseType}/tenant-id/ia/evaluate",
+        value = "/decision-definition/key/{decisionTableKey}/tenant-id/ia/evaluate",
```

Comment:
Just wondering about this.  So the decision table should be using the naming convention.  tablename-{jurisdiction)-{casetype}.  So if we have another service team we don't have to change any code and just use the same convention.  The tenant-id should also derived from the `jurisdiction` rather than hard-coded as `ia` since this would not work for another service.

Reply:
Are you getting the key using the `jurisdiction` and `caseType` before and then using that instead of the name?